### PR TITLE
Enable check feeding

### PR DESCRIPTION
**Pull request by pkajaba**
- Made chages in dockefile to enable downloading and building maven-index-checker
- Deleted and added patches (one which enables check feeding for maven)
- Pinned version of anitya until conflicts with patches are not resolved

### CODE REVIEW

File: patches/09-chech_feed-maven.patch [hunk 1]
```diff
@@ -0,0 +1,86 @@
+diff --git a/anitya/lib/backends/maven.py b/anitya/lib/backends/maven.py
```
Comment (bkabrda) [inline patches/09-chech_feed-maven.patch]: There's a typo in the name of this file (`chech` instead of `check`)

**Comment by bkabrda** [comment]
LGTM, just one minor nit - typo in filename of the patch. Will merge once you fix it.

File: patches/09-chech_feed-maven.patch (hunk 1 above)
Comment (pkajaba) [inline patches/09-chech_feed-maven.patch]: Oh, sorry I initially renamed it from previous typo `che_feed`.

Thanks!

**Comment by bkabrda** [comment]
LGTM, merging. Thanks!

**Pull request merged**